### ISSUE TITLE
Default parse-mode does not work for ctx.editMessageMedia

### ISSUE THREAD

**Issue by binamralamsal** [grammyjs/parse-mode#7]
Here is my code:
```javascript
await ctx.editMessageMedia(
  {
    media: new InputFile('file-location.png'),
    type: 'photo',
    caption: "<code>Test code here</code>",
  },
  { message_id: messageId },
);
```

It looks default parse-mode does not seem to work for this. I haven't tested with other edit method but it's not working with editMessageMedia for me.

It works fine if I set parse_mode explicitly.
```javascript
await ctx.editMessageMedia(
  {
    media: new InputFile('file-location.png'),
    type: 'photo',
    caption: "<code>Test code here</code>",
    parse_mode: 'HTML'
  },
  { message_id: messageId },
);
```

**Comment by KnightNiwrem** [comment]
This was (is?) somewhat intentional. The original purpose of the plugin was to ease migration from telegraf to grammy, and since there is no such support for edit in telegraf, there was no need to add it here either.

PQ to add to edit methods are welcomed, but I'd recommend using fmt here instead.

**Comment by KnorpelSenf** [comment]
Can this be closed since #14 is merged now?

**Comment by binamralamsal** [comment]
That's great to know!
